static/js/components/inputs/FieldsOfStudySelectField.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import _ from 'lodash';
+import SelectField from './SelectField';
+import FIELDS_OF_STUDY from '../../fields_of_study';
+
+let fieldOfStudyOptions = _.map(FIELDS_OF_STUDY, (name, code) => ({
+  value: code,
+  label: name
+}));
```

Comment:
I prefer using standard library stuff when possible over lodash - I think this is personal preference though, so if you feel strongly that this looks / feels better than something like this:

```js
Object.entries(FIELDS_OF_STUDY).map( ([k,v]) => ( ... ) );
```

feel free to keep what you have.